src/coreComponents/mainInterface/ProblemManager.cpp

```diff
@@ -621,55 +646,57 @@ map< std::pair< string, string >, localIndex > ProblemManager::calculateRegionQu
       for( localIndex a = 0; a < meshBodies.getSubGroups().size(); ++a )
```

Comment:
So, for now, the solver defines the discretization type and then applies it to all meshBodies? I wonder if at some point the same solver could work on different mesh bodies using different discretizations.